frontend/src/modules/details/api.ts

```diff
@@ -9,11 +9,16 @@ const fieldsParams = {
   format: 'geojson',
 };
 
+const fieldsParamsDetails = {
+  fields: `${fieldsParams.fields},ratings,ratings_description`,
+  format: 'geojson',
+};
+
 export const fetchDetails = (query: APIQuery, id: string): Promise<RawDetails> => {
   try {
-    return GeotrekAPI.get(`/trek/${id}/`, {
-      params: { ...query, ...fieldsParams, ...portalsFilter },
-    }).then(r => r.data);
+    return GeotrekAPI.get(`/trek/${id}/`, { params: { ...query, ...fieldsParamsDetails } }).then(
```

Comment:
Is there a reason why you deleted `...portalsFilter`?

Reply:
Oh no ! it's a copy paste error... Thanks for your attention